srcpkgs/gnome-initial-setup/template

```diff
@@ -18,4 +18,8 @@ license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Design/OS/InitialSetup"
 changelog="https://gitlab.gnome.org/GNOME/gnome-initial-setup/-/raw/master/NEWS"
 distfiles="${GNOME_SITE}/gnome-initial-setup/${version%%.*}/gnome-initial-setup-${version}.tar.xz"
-checksum=593cfc6dc8f82a98671b94c035b97dbe88ef56e708780b2ad1d2324e4d04bbff
+checksum=363e09aa3308e7f4074e066253a0182b322ab60376743de178bbb400e54ba8ee
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" dconf"
```

Comment:
Add this to hostmakedepends unconditionally.

Reply:
fixed